cdap-docs/reference-manual/source/http-restful-api-v3/namespace.rst

```diff
@@ -0,0 +1,142 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: HTTP RESTful Interface to the Cask Data Application Platform
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _http-restful-api-namespace:
+.. _http-restful-api-v3-namespace:
+
+===========================================================
+Namespace HTTP RESTful API
+===========================================================
+
+.. highlight:: console
+
+Use the CDAP Namespace HTTP API to create, list or delete namespaces in the CDAP instance.
+
+Namespaces, their use and examples, are described in the :ref:`Developers' Manual: Namespaces
+<namespaces>`.
+
+For the remainder of this API, it is assumed that the namespace you are using is defined
+by the ``<base-url>``, as descibed under :ref:`Conventions <http-restful-api-conventions>`. 
+
+Create a Namespace
+------------------
+To create a namespace, submit an HTTP PUT request::
+
+  PUT http://<host>:<port>/v3/<namespace-id>
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``<namespace-id>``
+     - Namespace ID
+
+The ``<namespace-id>`` must be of the limited character set for namespaces, as 
+described in the :ref:`Introduction <http-restful-api-namespace-characters>`.
+Properties for the namespace are passed in the JSON request body:
+
+.. list-table::
+   :widths: 20 40 40
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+     - Default Value (if not defined)
+   * - ``name``
+     - Display name for the namespace
+     - The Namespace ID
+   * - ``description``
+     - Display description of the namespace
+     - An empty string ("")
+
+In this initial release of namespaces, once a namespace has been created with a particular
+ID and properties, its properties cannot be edited. To change the display name and
+description for a particular ID, you need to delete the namespace and recreate it. A
+future release od CDAP will allow these properties to be edited.
+
+HTTP Responses
+..............
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Status Codes
+     - Description
+   * - ``200 OK``
+     - The event successfully called the method, and the namespace was created
+   * - ``409``
```

Comment:
OK, then the Google Doc needs updating, because that's what it says:
```
Create a namespace
URL: PUT <base-url>/v3/namespaces/<namespace-id>
Body: {“name”: “<name>”, “description”: “<description>”}
Response Status Codes:
200: Requested namespace was successfully created
400: Body contains invalid JSON or a valid JSON with null/empty name.
409: A namespace with the specified name already exists
500: Internal errors (mostly unchecked exceptions) within the service. a 500 could also occur if there is a problem reading the json content (throws IOException on the server side)
```